### ISSUE TITLE
【Core】【Bug】Unrecognized options: --resourceProfileId 0 --user-class-path file

### ISSUE THREAD

**Issue by allwefantasy** [byzer-org/kolo-lang#1643]
when byzer-lang runs on yarn client mode, the executors will produce many logs like following:

```
Unrecognized options: --resourceProfileId 0 --user-class-path file
```

The logs may occupy all disk space and crash the yarn cluster.

For now,  the user can walk around this issue by setting `-streaming.ps.cluster.enable` as false which is true by default.

This issue happens because in spark 3, spark add a new option in the executor start command, and we can parse new option and then print the logs we mentioned before.

**Comment by chncaesar** [comment]
Related Spark Issue https://issues.apache.org/jira/browse/SPARK-33462

**Comment by chncaesar** [comment]
Verified with hadoop 2.7.1 ( HDP 2.4.0 )
```shell
 $SPARK_HOME/bin/spark-submit --class streaming.core.StreamingApp \
        --driver-memory ${DRIVER_MEMORY} \
        --executor-memory 1g \
        --num-executors 1 \
        --jars ${JARS} \
        --master yarn \
        --deploy-mode client \
        --name mlsql \
        --conf "spark.sql.hive.thriftServer.singleSession=true" \
        --conf "spark.kryoserializer.buffer=256k" \
        --conf "spark.kryoserializer.buffer.max=1024m" \
        --conf "spark.serializer=org.apache.spark.serializer.KryoSerializer" \
        --conf "spark.scheduler.mode=FAIR" \
        ${MLSQL_HOME}/libs/${MAIN_JAR}    \
        -streaming.name mlsql    \
        -streaming.platform spark   \
        -streaming.rest true   \
        -streaming.driver.port 9004 
```

**Comment by chncaesar** [comment]
Close this issue. The fix is included [release-2.2.1](https://github.com/byzer-org/byzer-lang/releases/tag/v2.2.1) .
